src/freva/utils.py

```diff
@@ -176,7 +182,9 @@ def _hist(self) -> Dict[str, Any]:
         try:
             hist = cast(
                 List[Dict[str, Any]],
-                freva.history(entry_ids=self._id, return_results=True),
+                freva.history(
+                    entry_ids=self._id, return_results=True, user_name=self._user_name
+                ),
```

Comment:
Remember you've build in the `user-name` feature and that users can query `all` users. I think we can make use of that feature here: The history ids are unique and since the user already knows the id of a run from another user there is not much need in telling the thing again what user was involved with this id. Look at this (I am logged on as me):

```
(ssh) k204230@levante1 : ~
freva history --entry-ids  933409 --user-name 'all'
933409) animator [2024-06-26 10:57:10] finished {"input_file": "/work/bm1159/XCES/xces-work/b324056/MYWORK/postag/y...
```

So in think you simply have to set `user_name ` in the `freva.history` call to "*" or "all" and you are done. No need for another `user_name` parameter.